h2o-persist-s3/src/main/java/water/persist/PersistS3.java

```diff
@@ -64,13 +117,23 @@ public static AmazonS3 getClient() {
    * credentials provider.
    */
   public static class H2OAWSCredentialsProviderChain extends AWSCredentialsProviderChain {
-    public H2OAWSCredentialsProviderChain() {
-      super(new H2OArgCredentialsProvider(),
-          new InstanceProfileCredentialsProvider(),
-          new EnvironmentVariableCredentialsProvider(),
-          new SystemPropertiesCredentialsProvider(),
-          new ProfileCredentialsProvider());
+    public H2OAWSCredentialsProviderChain(final String accessKeyId, final String accessSecretKey) {
+      super(getProviders(accessKeyId, accessSecretKey));
     }
+
+    private static List<AWSCredentialsProvider> getProviders(final String accessKeyId, final String accessSecretKey) {
+      final List<AWSCredentialsProvider> providers = new ArrayList<>();
+      if (accessKeyId != null && accessSecretKey != null) {
+        providers.add(new AWSStaticCredentialsProvider(new BasicAWSCredentials(accessKeyId, accessSecretKey)));
+      }
+      providers.add(new H2OArgCredentialsProvider());
+      providers.add(InstanceProfileCredentialsProvider.getInstance());
+      providers.add(new EnvironmentVariableCredentialsProvider());
+      providers.add(new SystemPropertiesCredentialsProvider());
+      providers.add(new SystemPropertiesCredentialsProvider());
```

Comment:
it's the same as line above, copy error?

Reply:
Oh yes, a copy error ! Thank you ! This needs to be resolved !